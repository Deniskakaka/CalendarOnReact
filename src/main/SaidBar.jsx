import React from "react";
import { twentyFour } from "./functionTwentyFour.js";

const SaidBar = () => {
  return (
    <div className="saidBar">
      {twentyFour().map(i => (
        <div className="hourTime" key={i}>
          {i < 10 ? "0" + i + ":" + "00" : i + ":" + "00"}
        </div>
      ))}
    </div>
  );
};

export default SaidBar;
