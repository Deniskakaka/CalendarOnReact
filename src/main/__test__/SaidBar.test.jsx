import React from 'react';
import { shallow } from 'enzyme';
import SaidBar from '../SaidBar.jsx';


describe('<SaidBar/>', () => {
    it('should display class hour', () => {
        const wrappedComponent = shallow(<SaidBar/>)
        expect(wrappedComponent.find('.saidBar')).toMatchSnapshot(true);
    });
});

