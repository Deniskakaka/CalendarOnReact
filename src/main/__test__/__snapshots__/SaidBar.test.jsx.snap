// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SaidBar/> should display class hour 1`] = `
<div
  className="saidBar"
>
  <div
    className="hourTime"
    key="0"
  >
    00:00
  </div>
  <div
    className="hourTime"
    key="1"
  >
    01:00
  </div>
  <div
    className="hourTime"
    key="2"
  >
    02:00
  </div>
  <div
    className="hourTime"
    key="3"
  >
    03:00
  </div>
  <div
    className="hourTime"
    key="4"
  >
    04:00
  </div>
  <div
    className="hourTime"
    key="5"
  >
    05:00
  </div>
  <div
    className="hourTime"
    key="6"
  >
    06:00
  </div>
  <div
    className="hourTime"
    key="7"
  >
    07:00
  </div>
  <div
    className="hourTime"
    key="8"
  >
    08:00
  </div>
  <div
    className="hourTime"
    key="9"
  >
    09:00
  </div>
  <div
    className="hourTime"
    key="10"
  >
    10:00
  </div>
  <div
    className="hourTime"
    key="11"
  >
    11:00
  </div>
  <div
    className="hourTime"
    key="12"
  >
    12:00
  </div>
  <div
    className="hourTime"
    key="13"
  >
    13:00
  </div>
  <div
    className="hourTime"
    key="14"
  >
    14:00
  </div>
  <div
    className="hourTime"
    key="15"
  >
    15:00
  </div>
  <div
    className="hourTime"
    key="16"
  >
    16:00
  </div>
  <div
    className="hourTime"
    key="17"
  >
    17:00
  </div>
  <div
    className="hourTime"
    key="18"
  >
    18:00
  </div>
  <div
    className="hourTime"
    key="19"
  >
    19:00
  </div>
  <div
    className="hourTime"
    key="20"
  >
    20:00
  </div>
  <div
    className="hourTime"
    key="21"
  >
    21:00
  </div>
  <div
    className="hourTime"
    key="22"
  >
    22:00
  </div>
  <div
    className="hourTime"
    key="23"
  >
    23:00
  </div>
</div>
`;
